### DISCUSSION TITLE
Monthly ESMValTool community meeting October 2023

### DISCUSSION THREAD

**Discussion by paultgriffiths** [ESMValGroup/Community#125] [Meetings]
Dear @ESMValGroup/esmvaltool-developmentteam,

The next online community meeting will be on

October 16th at 3pm CEST / 2pm BST

The duration will be one hour.   [Join the zoom meeting](https://ncas.zoom.us/j/98906539291?pwd=Nkh1R0JqSzNHQlFmdmFTNHd2bXNzQT09).

Calendar invitations to the meeting are sent to the mailing list, join it [here](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list).

**Format and agenda**

- Anyone with an interest in ESMValTool is welcome to join and ask questions

- We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.

- Using the input from the introduction round, we choose some topics to talk about in more detail.

- If you already know that you would like to bring up a particular topic in the meeting, you can mention the GitHub issue on the topic in a comment below. That will make it easy to find for the other people in the meeting and we can already start discussing there before the meeting. Just create a new GitHub issue about your topic if there is no open issue yet.

- GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the previous meeting are available in #105

**Comment by ehogan** [comment]
Would it be possible to discuss if / when the next workshop will be scheduled, please? 😊

**Reply by paultgriffiths** [reply]
sure thing - what would you like to add?

**Reply by ehogan** [reply]
Thanks @paultgriffiths! Nothing specific, just an open question of whether anyone in the community would like to host the next workshop 😊

**Comment by malininae** [comment]
Oh, I'd like to discuss a few things on preprocessor (e.g. https://github.com/ESMValGroup/ESMValCore/issues/1895 ) and give update on what I want to submit a pull request for in the foreseeable future.

**Comment by glpotter** [comment]
I would like to hear about the progress of producing cmorizers for reanalyses.

**Comment by paultgriffiths** [comment]
Updates:
- @ehogan date/location of workshop. @rswamina Poll for dates/location is coming
- @malininae how to add developers?  email @axel-lauer with GH username to add to team.
- @malininae have a package that depends on R and Python (Python package rather fiddly to install). How to add to ESMValTool?  @bouweandela if license permits, open an issue to start the process
- @malininae How to weight data by number of realisations?  @bouweandela suggest use [climwip](https://docs.esmvaltool.org/en/latest/recipes/recipe_climwip.html) 
- @glpotter CMORisers for reanalysis.  @zklaus will be in contact to liaise.
- @malininae derivation of 3hr windspeed from ERA data.  Possible to use preprocessor to do this?
- @egalytska [Fresh Eyes on CMIP](https://wcrp-cmip.org/fresh-eyes-on-cmip-launch/) is proposing an EGU session with Hackathon.  Sessions announced November 1st. @rswamina is in the process of preparing a video for the CMIP IPO as part of the [CMIP7 Model Benchmarking Team](https://wcrp-cmip.org/cmip7-task-teams/model-benchmarking/)

**Reply by valeriupredoi** [reply]
@paultgriffiths that's brilliant you took over organizing this, many thanks and :beer: Could you please send a calendar invite next time, once the date and time of the call is set? I've not received one for yesterday's call :cry:

**Reply by paultgriffiths** [reply]
Hi @valeriupredoi - do you mean attach an invite (ics) to the mail I send to ESMValTool@listserv.dfn.de?  Wasn't sure of the etiquette!

**Reply by valeriupredoi** [reply]
shoot! Sorry, mate - I totally left this one to the elements. Yes, pls send that thingie where there is a calendar entry where I press "yes" and that puts it in my calendar :grin:
